### ISSUE TITLE
Default query has become too heavy for Virtuoso after system update

### ISSUE THREAD

**Issue by roland-c** [dvcama/LodView#32]
Hi,
I had to change the default query in conf.ttl and had to comment the last 2 UNIONs out because of sudden performance problems of Virtuoso. After updating the several Ubuntu systems LodView became unresponsive for resources containing a certain nr of triples. Resources containing only a few triples were resolved well but more elaborate resources hung the LodView interface. 
Apparently the Virtuoso response became too long and generated a 500 error, causing LodView to fail. After simplifying the query this error didn't appear anymore.

I think this is an issue caused by performance reduction of patched Intel based system because of the Meltdown/Spectre flaws. I'm not sure and would like to know if anyone else ran into performance issues after system updates.

tl;dr; When LodView is hung after system update check the default query, it may need simplification.
